knowledge-base/kb-window-howto-use-radwindow-as-mainwindow

```diff
@@ -0,0 +1,81 @@
+---
+title: How to Use RadWindow as MainWindow
```

Comment:
The same feedback here as the one in the other KB review I posted.
https://github.com/telerik/xaml-docs/pull/1561